renderer.js

```diff
@@ -355,7 +367,45 @@ function onVersionDataReceived(data, unsigned = false){
     });
 }
 
-function loadVersionData(){
+// Checks the graphics card
+async function checkIfCompatible() {
+    try {
+        const data = await si.graphics();
+        const identifier = ["nvidia", "advanced micro devices", "amd"]; // and so on...
+
+        let cardsVendor = [];
+
+        for(var i = 0; i < data.controllers.length; i++){
+			
+            let graphicControllers = data.controllers[i];
+			
+            cardsVendor.push(graphicControllers.vendor.toLowerCase());
+            cardsModel.push(" " + graphicControllers.model);
+            
+            for(var n = 0; n < cardsVendor.length; n++){
```

Comment:
This loop is totally unnecessary as each vendor / card is already looped through by the top level loop (starting at line 378).